Week4/app/src/main/java/com/em4n0101/mymovies/MoviesAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package com.em4n0101.mymovies
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.em4n0101.mymovies.data.Movie
+import com.em4n0101.mymovies.utils.MovieViewHolder
+
+class MoviesAdapter(private val movieList: ArrayList<Movie>, val delegate: SelectItemListener): RecyclerView.Adapter<MovieViewHolder>() {
+
+    interface SelectItemListener {
+        fun listItemPressed(list: Movie)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieViewHolder {
+        val holder = LayoutInflater.from(parent.context)
```

Comment:
`LayoutInflater.from()` doesn't return holder instance, it returns View. Thus, this naming is not really accurate